docs/wmco-prerequisites.md

```diff
@@ -6,11 +6,17 @@ may be relevant.
 | Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
 |---------------------------|---------------------------|------------------------|
 | Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
-| Azure                     | 4.6+                      | WMCO 1.0+              |
+| Azure                     | 4.11+                     | WMCO 6.0+              |
 | GCP                       | 4.12+                     | WMCO 7.0+              |
 | VMware vSphere            | 4.7+                      | WMCO 2.0+              |
 | Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
```

Comment:
Thoughts on making the table more chronological?
```
| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
|---------------------------|---------------------------|------------------------|
| Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
| VMware vSphere            | 4.7+                      | WMCO 2.0+              |
| Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
| Azure                     | 4.11+                     | WMCO 6.0+              |
| GCP                       | 4.12+                     | WMCO 7.0+              |
```

Reply:
I could but that would have made reviewing this PR more difficult. But I can do this.

Reply:
Done.